src/cli_util.rs

```diff
@@ -1221,6 +1221,26 @@ pub fn write_commit_summary(
     Ok(())
 }
 
+pub fn write_config_entry(
```

Comment:
Another thing here: Feels like this serialization helper might eventually belong on Config? Hoping it's okay stashing it here for now.

Reply:
Yes, this seems like a good place. For serializing config into TOML, I hope some TOML library will help us do that instead.

Reply:
> For serializing config into TOML, I hope some TOML library will help us do that instead.

`cargo add` appears to use `toml_edit`. Maybe we can try it.
https://crates.io/crates/toml_edit

Reply:
Good find. The limitations mentioned there are kind of big but probably fine. The API looks very easy to work with.

Reply:
Added a TODO.